saleor/order/migrations/0164_auto_20230310_1118.py

```diff
@@ -0,0 +1,61 @@
+# Generated by Django 4.0.7 on 2022-09-26 10:15
+
+from django.db import migrations, models
+
+
+def rename_order_events(apps, _schema_editor):
+    OrderEvent = apps.get_model("order", "OrderEvent")
+    OrderEvent.objects.filter(type="transaction_void_requested").update(
+        type="transaction_cancel_requested"
+    )
+    OrderEvent.objects.filter(type="transaction_capture_requested").update(
+        type="transaction_charge_requested"
+    )
```

Comment:
We should ensure 0 downtime here cc. @kadewu

Reply:
Comment: we are going to re-do this migration in the next version after the release, instead of the previous version patch approach, as we don't want to change `Event.type` in the previous version.

Reply:
Ok, but the `update` will lock the whole table AFAIK, and it should be improved.

Reply:
Yes right, it should be run in batches. Thanks!